src/components/DocumentationTopic.vue

```diff
@@ -40,7 +41,12 @@
             :data-tag-name="tagName"
           />
         </Title>
-        <Abstract v-if="abstract" :content="abstract" />
+        <LinkableHeading v-else class="minimized-summary">Summary</LinkableHeading>
```

Comment:
To make this a little logic slightly more straightforward to read, you may want to consider moving this before `Title` and swapping the conditions.

Instead of:
```
Title if not minimized
Summary else
```

Something like:
```
Summary if minimized
Title else
```

I would also consider just using the title component for this and changing the text but it sounds like these may be using a different heading level which makes that more complex and probably not worth looking into.

Reply:
Good point! I put `Summary` second because it's a less frequent scenario. I can see how that's confusing and not straightforward. I fixed it.

> I would also consider just using the title component for this and changing the text but it sounds like these may be using a different heading level which makes that more complex and probably not worth looking into.

Yup, the `Summary` heading is meant to be the same level as all the headers for primary content. I will keep this in mind for the future.